src/boot/loader.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { Modal, Padding, Text } from '@zextras/carbonio-design-system';
+import React, { FC, useCallback, useEffect, useState } from 'react';
```

Comment:
style: unused import
```suggestion
import React, { useCallback, useEffect, useState } from 'react';
```